Sources/Plasma/FeatureLib/pfMoviePlayer/plMoviePlayer.h

```diff
@@ -72,7 +72,9 @@ class plMoviePlayer
     class plMipmap* fTexture;
 
     mkvparser::MkvReader* fReader;
+#ifdef USE_WEBM
```

Comment:
Why not put fReader in here too?

Reply:
It's only ever dereferenced if `USE_WEBM`, but it is unconditionally initialized. `std::unique_ptr`, OTOH, doesn't like deleting an forward-declared type whose value is null.

Reply:
Yeah, I get why `fSegment` is protected...  But my point is that every other use of `fReader` other than the initializer is already protected, so why not fix the initializer too?

Reply:
True. Another compelling solution would be to just get rid of all of this `USE_WEBM` stuff -- back when this was written, building libvpx was nontrivial, but vcpkg makes it trivial, if time consuming.